lib/nexmo_developer/app/views/static/landing.html.erb

```diff
@@ -17,30 +17,30 @@
             <h2>
               <a href="/tools">
                 <div><svg class="Vlt-icon Vlt-icon--large Vlt-purple-dark"><use xlink:href="/symbol/volta-icons.svg#Vlt-icon-design-tools"/></svg></div>
-                SDKs &amp; Tools
+                Featured Topic 1
               </a>
             </h2>
-            <p>The Nexmo libraries allow you to get up and running with<br>Nexmo APIs quickly in your language of choice.</p>
+            <p>You can choose to highlight an area of your portal on the landing page.</p>
           </div>
         </div>
         <div class="Vlt-col Vlt-col--M-1of2">
           <div>
             <h2>
               <a href="/community">
                 <div><svg class="Vlt-icon Vlt-icon--large Vlt-purple-dark"><use xlink:href="/symbol/volta-icons.svg#Vlt-icon-group"/></svg></div>
-                Community
+                Featured Topic 2
               </a>
             </h2>
-            <p>Find out about our talks, community hacks,<br>and what events we'll be at.</p>
+            <p>You can highlight more than one area.</p>
           </div>
         </div>
       </div>
 
       <hr class="hr--tall">
 
       <div class="center">
-        <h3>Do you have a question?</h3>
-        <p class="p-large">We're always here to help. Check out our <a href="https://help.nexmo.com">support FAQs</a>, contact our super helpful <a href="https://help.nexmo.com/hc/en-us/requests/new">support team</a> or get in touch on <a href="http://stackoverflow.com/questions/tagged/nexmo">Stack Overflow</a>.<br />Your data will be treated in accordance with our <a href="https://www.nexmo.com/privacy-policy">Privacy Policy</a>, which sets out the rights you have in respect of your data.</p>
+        <h3>Modifying This Content</h3>
+        <p class="p-large">To modify this content you can copy this file to the `custom/views/static` folder in your documentation folder and modify that copy. More detailed instructions to create your own landing page can be found on the <a href="https://nexmo.github.io/station/How-To-Use#customization-options">Station documentation</a>.</p>
```

Comment:
How do they know what this file is if they're seeing a rendered page? Should it link to the view on GitHub?

Reply:
Ah, yes. We can do that.